tools/mkdfs/mkdfs.c

```diff
@@ -1,20 +1,30 @@
 #include <stdio.h>
-#include <stdlib.h>
-#include <dirent.h>
+#include <malloc.h>
 #include <sys/stat.h>
 #include <string.h>
 #include <errno.h>
 #include <stdint.h>
+
 #include <sys/types.h>
+#ifndef _MSC_VER
+#include <dirent.h>
 #include <sys/param.h>
+#else
+#include "dirent.h"
```

Comment:
How is this resolved on vs? If you have a local file, maybe we should also include that.

Reply:
As per the PR description...

Reply:
Ah now seen it sorry about that. Still we may need to include that info somewhere visible

Reply:
I agree, would a readme in the folder be acceptable, including the compiler command for MSVC i.e.       
call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
      cl .\tools\mkdfs\mkdfs.c -I .\include\

Reply:
I think that’ll be fine

Reply:
Readme added. I guess I should check that malloc is still needed, so please hold off on committing until I am able to test...